pkg/management/component.go

```diff
@@ -142,13 +151,14 @@ type component struct {
 	usersDBModule           UsersDetailsDBModule
 	eventDBModule           database.EventsDBModule
 	configDBModule          keycloakb.ConfigurationDBModule
+	onboardingModule        keycloakb.OnboardingModule
```

Comment:
You don't need to copy/paste OnboardingModule interface in this file if you use keycloakb.OnboardingModule
In the NewComponent, you are using the local OnboardingModule.
You might choose which one you are using for this whole file

Reply:
good catch thks